src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/eventpicker/eventpicker.html

```diff
@@ -17,7 +17,7 @@
               <div class="umb-tree-item__inner" style="padding: 5px 10px;">
                 <button type="button" class="btn-reset text-left flex-auto"
                         ng-class="{ 'umb-tree-node-checked': event.selected }" ng-click="vm.selectEvent(event)">
-                  <umb-icon icon="{{event.selected ? 'icon-check' : 'icon-globe'}}" class="icon umb-tree-icon"
+                  <umb-icon icon="{{event.selected ? 'icon-check' : 'icon-flag'}}" class="icon umb-tree-icon"
```

Comment:
Searching for webhook many seem to use this icon:
https://fontawesome.com/icons/webhook?f=classic&s=solid

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/8344d105-2b46-449a-99ba-dd3af2879ec7)

Not sure if it is standard, but it could include a custom SVG icon.

Reply:
Yes I would love this! atleast for the Webhooks section, but I cannot for the life of my figure out how to add an icon 🙈

Reply:
Just put the SVG icon in the same folder as `Umbraco.svg` icon here 😁 
https://github.com/umbraco/Umbraco-CMS/blob/634afc4b18d2b1bfd5fbe6de290c9790b2218074/src/Umbraco.Web.UI.Client/src/assets/icons/icon-umbraco.svg?plain=1

If e.g. Umbraco Forms ships with custom icons they would be placed in the App_Plugins folder following this structure `/App_Plugins/MyPlugin/Backoffice/Icons/`:
https://our.umbraco.com/forum/using-umbraco-and-getting-started/106330-how-to-use-custom-svg-icons#comment-331354

Reply:
Maybe something like this icon: https://www.svgviewer.dev/s/403059/webhook

Reply:
I tried doing that, but I still couldn't do `icon-webhook` for example 🤔 
Maybe I just did it wrong 🙈

Reply:
Is the icon name `icon-webhook.svg` or just `webhook.svg`?

Reply:
icon name was icon-webhook.svg iirc 😅

Reply:
It should work, but the icons are cached, so it requires the application to be restart and maybe the browser cache to be cleared as well 🙂

Reply:
I got it to work!
I found this icon, but it looks really zoomed in for some reason, when applied as an icon 🙈 

![webhook (1)](https://github.com/umbraco/Umbraco-CMS/assets/70372949/e7dc8b34-6ac0-414d-a460-1e90ff272367)

Reply:
Yes, it is similar to the webhook icon in Lucide icon set
https://github.com/umbraco/Umbraco.UI/pull/629
https://lucide.dev/icons/webhook

Reply:
Yep thats the exact one I choose, but it looks wierdly zoomed in 🙈 
Might need a follow up PR if you're up for it, because I apparently suck at using icons 🤣

Reply:
I guess it depend on the viewbox size or it width and height attributes has been set, but we can always look into that before final release 😎

Reply:
Yes please, I am starting to loose hair 🤣